mate-base/mate-settings-daemon/files/mate-settings-daemon-1.26.0-fix-buid-with-clang16.patch

```diff
@@ -0,0 +1,14 @@
+Bug: https://bugs.gentoo.org/881315
```

Comment:
You've sent it upstream, I assume? If so, where's the link? If not, why not? 😉 

Always upstream!